dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -223,7 +303,18 @@ public WorkerBenchTaskResult runLocal() throws Exception {
 
   @Override
   public void validateParams() throws Exception {
-    //no-op
+    if (mBaseParameters.mClusterLimit <= 0) {
+      throw new IllegalStateException("--cluster-limit cannot be " + mBaseParameters.mClusterLimit
+          + " in StressWorkerBench. It should be a positive number. Use 1 if running in local mode");
+    }
```

Comment:
We used to allow `--cluster-limit == 0 ` and `<0`. I don't think that makes much sense.